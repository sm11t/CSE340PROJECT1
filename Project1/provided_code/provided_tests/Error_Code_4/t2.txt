TASKS
1 2
POLY
    F1 = x + 2;
    F2(X,Y,Z) = X + Y + Z;
    F3(X) = X^2 + 1;
    F4(X,Y) = X + Y;
EXECUTE
    INPUT W;
    a = F2(W,W);
    b = F3(a,W);
    c = F4(a,b);
    d = F1(b,c);
INPUTS
1 2 3 4 5