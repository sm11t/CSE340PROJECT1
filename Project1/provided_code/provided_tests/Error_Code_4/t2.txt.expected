Semantic Error Code 4: 10 11 13 
